src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationScheme.cs

```diff
@@ -2,32 +2,33 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Reflection;
-using Microsoft.AspNetCore.Http;
 
```

Comment:
@Tratcher they are here, but perhaps hard to find in the 178 diff...  maybe easier to just view via commits